esmvaltool/diag_scripts/shared/plot/scatterplot.ncl

```diff
@@ -493,12 +497,28 @@ begin
   end if
 
   ; Check if a valid wks has been provided, otherwise invoke default
-  wks = get_wks(wks_in, diag_script, var)
+  wks = get_wks(wks_in, DIAG_SCRIPT, var)
 
   ; Define style
-  annots = project_style(diag_script_info, "annots")
-  colors  = project_style(diag_script_info, "colors")
-  markers = project_style(diag_script_info, "markers")
+; A_laue_ax+
```

Comment:
I would remove this notation, using git is no longer necessary to manually mark changes.